### PR TITLE
fix: do not export ama-sdk-core as es module by default

### PR DESCRIPTION
**Pull request by mrednic-1A**
## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

File: packages/@ama-sdk/core/project.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
         "script": "build:esm2015"
       },
       "dependsOn": [
-        "compile"
```
Comment (kpanot) [inline packages/@ama-sdk/core/project.json:30]: Why did you change the default `compile` name which comes with default config?

**Review by kpanot — APPROVED** [review]

File: packages/@ama-sdk/core/project.json (hunk 1 above)
Comment (mrednic-1A) [reply packages/@ama-sdk/core/project.json:30]: There is no technical reason behind the change. I changed it to be aligned on naming with the other configs.

File: packages/@ama-sdk/core/package.json [hunk 2]
```diff
@@ -29,9 +29,9 @@
     "build": "yarn nx build ama-sdk-core",
     "build:cjs": "swc src -d dist/cjs -C module.type=commonjs -q",
     "build:esm2015": "swc src -d dist/esm2015 -C module.type=es6 -q",
-    "build:esm2020": "tsc -b tsconfig.esm2020.json",
+    "build:esm2020": "tsc -b tsconfig.build.json",
```
Comment (vscaiceanu-1a) [inline packages/@ama-sdk/core/package.json:32]: `tsconfig.esm2020.json` still exist. Is is used somewhere else or can we remove it?

**Review by vscaiceanu-1a — APPROVED** [review]

**Review by fpaul-1A — APPROVED** [review]

File: packages/@ama-sdk/core/package.json (hunk 2 above)
Comment (mrednic-1A) [reply packages/@ama-sdk/core/package.json:32]: No, the file does not exist. The reason why we got nothing before is that the script `build:esm2020` was never called. (this build was done by the `compile` configuration which used the nx/js:tsc builder - which added the type: module in package.json btw. So I removed nx builder and align the naming with other builds and call this script defined in package.json)

Comment (vscaiceanu-1a) [reply packages/@ama-sdk/core/package.json:32]: I understand. Thanks!
